### PR TITLE
auto brewery symptom

### PR DESCRIPTION
**Pull request by nikk735**
## About The Pull Request

adds auto brewery symptom

## Why It's Good For The Game

just another virus symptom, thought it was interesting
can be annoying if you get unlucky with the ticks probably wont kill you though 

## Changelog
:cl:
added the symptom
/:cl:

### CODE REVIEW

**Comment by zeskorion** [comment]
1. Fix up your branch,  you should use a different branch from spiked skin
2. I'd advise changing your formulas. This procs every 1-5 ticks? That's invariably deadly if you have a liver
3.maybe make it actually make you drunk dirextly instead of synthesizing booze

**Comment by nikk735** [comment]
I could make it add drunkness yes but that’s not as authentic imo. Plus you can’t eat charcoal pills to help and etc.

**Comment by knacker48** [comment]
Huh, thats a real life condition https://en.wikipedia.org/wiki/Auto-brewery_syndrome

**Comment by nikk735** [comment]
@knacker48 lmao yea thats what i read too i thought it was a joke at first

**Comment by zeskorion** [comment]
> I could make it add drunkness yes but that’s not as authentic imo. Plus you can’t eat charcoal pills to help and etc.

doesn't help with the fact the thresholds barely do shit, and the tickspeed is all wonky. if you really wanted authenticity, you'd make it convert nutriment

**Comment by nikk735** [comment]
no it doesn’t lol but it’s not that bad I can make it more consistently if you’d like

**Comment by zeskorion** [comment]
make it rely less on rng, lower the values but make the thresholds more impactful, especially because bee booze causes confusion

**Comment by nikk735** [comment]
Ok will do soon

File: code/datums/diseases/advance/symptoms/brew.dm [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	stage_speed = -5
+	transmittable = -2
+	level = 5 // feel free to change
+	symptom_delay_min = 1
+	symptom_delay_max = 5
+	severity = 3
+	base_message_chance = 5
+	var/Power = 1
+	var/dangerous = FALSE
+	var/mob/living/C = null
```
Comment (ike709) [inline code/datums/diseases/advance/symptoms/brew.dm:28]: Unused?

File: code/datums/diseases/advance/symptoms/brew.dm [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	var/mob/living/carbon/affected_mob = null
```
Comment (ike709) [inline code/datums/diseases/advance/symptoms/brew.dm:29]: This is never set.

File: code/datums/diseases/advance/symptoms/brew.dm [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+	var/mob/living/M = A.affected_mob
+
+	if(!..())
+		return
+
+	if(prob(6.66*Power*((A.stage/10)+1)))
+		if (dangerous == TRUE)
+			M.reagents.add_reagent(/datum/reagent/consumable/ethanol, 12)
+			to_chat(M, "<span class='warning'>You feel your stomach bubbling aggressively.</span>")
+		else
```
Comment (ike709) [inline code/datums/diseases/advance/symptoms/brew.dm:50]: There isn't much difference between these two.

File: code/datums/diseases/advance/symptoms/brew.dm (hunk 2 above)
Comment (ike709) [reply code/datums/diseases/advance/symptoms/brew.dm:29]: Also isn't `affected_mob` already a var on symptoms?

**Pull request closed without merging**